modules/user-access/pages/enabling-user-authentication.adoc

```diff
@@ -56,7 +56,7 @@ To enable RESTPP authentication, set the `RESTPP.Factory.EnableAuth` parameter t
 
 . As the TigerGraph Linux user, run the following command:
 +
-.Enabling REST{pp} SAML2.0 Authentication
+.Enabling REST{pp} Authentication
```

Comment:
I wonder if this change is expected as it's not part of the original PR #378

Reply:
You are correct! That change was expected. It is part of another ticket I was working on. It showed up and decided to get this out of the way at the same time: https://graphsql.atlassian.net/browse/DOC-1855 than you for bringing it up it slipped my mind to mention it.